wweval/R/combine_outputs.R

```diff
@@ -66,11 +67,41 @@ combine_outputs <- function(output_type =
         },
         error = function(e) {}
       )
+    } else {
+      # Create a tibble of the combos that are missing, to save
+      this_failed_output <- tibble(
+        scenario = this_scenario,
+        location = this_location,
+        forecast_date = this_forecast_date
+      )
+      flag_failed_output <- rbind(flag_failed_output, this_failed_output)
     }
   }
 
   if (nrow(combined_output) == 0) {
     combined_output <- NULL
   }
+
+  if (nrow(flag_failed_output) != 0) {
+    warning(glue::glue(
+      "File missing for {this_scenario}",
+      "in {this_location} on {this_forecast_date}"
+    ))
```

Comment:
I believe that, as written, this will always warn about the final scenario/location/date processed, versus warning about any individual flagged trios. To fix this, the actual warning should happen in the for-loop, whereas the table of files missing can be written out at the end.